src/abrt-checker.c

```diff
@@ -674,18 +674,18 @@ static void write_to_cel(
 
     char *bt_escaped_newline = escape_newline_chars(backtrace);
 
-    char *json = xasprintf("{\"%s\": \"%s\", "
-                            "\"%s\": \"%s\", "
-                            "\"%s\": \"%s\", "
-                            "\"%s\": \"%s\", "
-                            "\"%s\": \"%s\", "
-                            "\"%s\": \"%s\"}\n",
-                            FILENAME_TYPE, FILENAME_TYPE_VALUE, /* type */
-                            FILENAME_EXECUTABLE, executable, /* executable */
-                            FILENAME_REASON, message, /* reason */
-                            FILENAME_BACKTRACE, bt_escaped_newline, /* backtrace */
-                            FILENAME_UID, uid, /* uid */
-                            "abrt-java-connector", VERSION);
+    char *json = libreport_xasprintf("{\"%s\": \"%s\", "
```

Comment:
Suggestion: use `g_strdup_printf()`. We want to get rid of these helper functions, anyway. And, since, there already seems to be a transitive dependency on GLib, this wouldn’t add anything new.

Reply:
Replaced.